_includes/footer.html

```diff
@@ -6,9 +6,9 @@
                 <p>Made with ❤️ by the Vanilla OS Contributors.</p>
             </div>
             <div class="footer-area">
-                <p>Vanilla OS is a Free and Open Source project. <a href="{{ site.url }}/get-involved">Learn here</a> how to get involved.</p>
+                <p>Vanilla OS is a free and open source project. <a href="{{ site.url }}/get-involved">Learn here</a> how to get involved.</p>
                 <p>
-                    <small>Every software and component is licensed under different Open Source licenses.</small>
+                    <small>Every software and component is licensed under different open source licenses.</small>
```

Comment:
```suggestion
                    <small>Every software and component is licensed under different open-source licenses.</small>
```

Reply:
I see "open source" significantly more often than "open-source", any particular reason to use it?

Reply:
> I see "open source" significantly more often than "open-source", any particular reason to use it?

Check this [out](https://www.computerhope.com/jargon/o/opensour.htm#:~:text=Both%20%22open%20source%22%20and%20%22,writing%20to%20help%20prevent%20confusion.) and [this](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source) based on context it is hyphenated, in this context I think hypenating it is better.

Reply:
I will check this PR later tomorrow again. It's late here 😄 .

Reply:
Ah, yeah I see that now. On those sites, open source was being used as a noun. I never knew about this convention, I'll merge these